src/test/java/net/sf/jabref/event/AddEntryEventTest.java

```diff
@@ -0,0 +1,32 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+import com.google.common.eventbus.EventBus;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AddEntryEventTest {
+
+    private EventBus eventBus;
+    private TestEventListener testEventListener;
+
+    @Before
+    public void setUp() {
+        eventBus = new EventBus();
+        testEventListener = new TestEventListener();
+        eventBus.register(testEventListener);
+    }
+
+    @Test
+    public void testEventReceivement() {
+        BibEntry shouldBeBibEntry = new BibEntry();
+        shouldBeBibEntry.setId("testkey1");
+        AddEntryEvent aocee = new AddEntryEvent(shouldBeBibEntry);
+        eventBus.post(aocee);
+
+        Assert.assertEquals(shouldBeBibEntry.getId(), testEventListener.getBibEntry().getId());
+        Assert.assertEquals(shouldBeBibEntry.getId(), aocee.getBibEntry().getId());
```

Comment:
What is the test failure message here? Did you tried out the with the latest master branch because the BibtexEntryAssert was changed recently. There is also an open PR by Olly about this.